tests/Feature/Groups/GroupDeleteTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Feature\Groups;
+
+use App\Group;
+use App\Role;
+use Tests\TestCase;
+
+class GroupDeleteTest extends TestCase {
```

Comment:
Should we also have a test for the check regarding not being able to delete if past events / devices exist?

Reply:
Yes, and adding that finds bugs.  Tests do that.  Can you rereview?